channels/id.m3u

```diff
@@ -103,8 +103,6 @@ https://video.detik.com/transtv/smil:transtv.smil/playlist.m3u8
 http://edge.vediostream.com/abr/tvikim/live/tvikim_480p/playlist.m3u8
 #EXTINF:-1 tvg-id="TVKU.id" tvg-name="TVKU" tvg-country="ID" tvg-language="Indonesian" tvg-logo="https://m3u-editor.com/storage/channel-logos/07cd9d90-3cc5-11ea-9aa8-0b5bd3ba261d/92697/14040332.png" group-title="",TVKU
 http://103.30.1.14:8080/hls/live.m3u8
-#EXTINF:-1 tvg-id="TVRParlemen.id" tvg-name="TVR Parlemen" tvg-country="ID" tvg-language="Indonesian" tvg-logo="https://yt3.ggpht.com/ytc/AAUvwnhlaM4KZ8xQaJpfP8GPX7Ji3tYXVnuSVfVKNV69=s900-c-k-c0x00ffffff-no-rj" group-title="Legislative",TVR Parlemen
-http://103.18.181.69:1935/golive/livestream/playlist.m3u8
```

Comment:
This might be the type of channel to go Not 24/7 when no parlementary sessions are open.

Reply:
Putting it on Not 24/7, to look out.